src/pages/ProjectCreation/ProjectCreation.context.tsx

```diff
@@ -0,0 +1,267 @@
+import useMutationAlert from "src/api/useMutationAlert";
+import ProjectApi from "src/api/Project";
+import { generatePath, useNavigate, useSearchParams } from "react-router-dom";
+import { RoutePaths } from "src/App";
+import { AutoSaveForm } from "src/hooks/useAutoSave/AutoSaveForm";
+import { STORAGE_KEY_CREATE_PROJECT_FORM, useResetStorage } from "./hooks/useProjectCreationStorage";
+import { onSyncOrganizations } from "./utils/syncOrganization";
+import { watchInstalledRepoStorage } from "./utils/watchInstalledRepoStorage";
+import { StorageInterface } from "src/hooks/useStorage/Storage";
+import MeApi from "src/api/me";
+import { UseGithubOrganizationsResponse } from "src/api/me/queries";
+
+/**
+ * @interface CreateContextProps
+ * @property {CreateFormData | undefined} initialProject - Initial project data.
+ * @property {ProjectCreationSteps | undefined} initialStep - Initial step in project creation.
+ * @property {number[] | undefined} initialInstalledRepo - Initially installed repositories.
+ * @property {React.ReactNode} children - React components.
+ * @property {StorageInterface<CreateFormData | undefined>} formStorage - Storage for form data.
+ * @property {StorageInterface<ProjectCreationSteps>} stepStorage - Storage for step information.
+ * @property {StorageInterface<number[]>} reposStorage - Storage for repository information.
+ */
+interface CreateContextProps {
+  initialProject: CreateFormData | undefined;
+  initialStep: ProjectCreationSteps | undefined;
+  initialInstalledRepo: number[] | undefined;
+  children: React.ReactNode;
+  formStorage: StorageInterface<CreateFormData | undefined>;
+  stepStorage: StorageInterface<ProjectCreationSteps>;
+  reposStorage: StorageInterface<number[]>;
+}
+
+type CreateProject = {
+  form: UseFormReturn<CreateFormData, unknown>;
+  currentStep: ProjectCreationSteps;
+  installedRepos: number[];
+  organizations: UseGithubOrganizationsResponse[];
+  formFn: {
+    addRepository: (data: CreateFormDataRepos) => void;
+    removeRepository: (data: CreateFormDataRepos) => void;
+  };
+  helpers: {
+    saveInSession: () => void;
+    goTo: (step: ProjectCreationSteps) => void;
+    next: () => void;
+    prev: () => void;
+  };
+};
+
+export const CreateProjectContext = createContext<CreateProject>({
+  form: {} as UseFormReturn<CreateFormData, unknown>,
+  currentStep: ProjectCreationSteps.ORGANIZATIONS,
+  installedRepos: [],
+  organizations: [],
+  helpers: {
+    saveInSession: () => null,
+    goTo: () => null,
+    next: () => null,
+    prev: () => null,
+  },
+  formFn: {
+    addRepository: () => null,
+    removeRepository: () => null,
+  },
+});
+
+const validationSchema = z.object({
+  logoUrl: z.string().optional(),
+  inviteGithubUserIdsAsProjectLeads: z.array(z.number()).optional(),
+  isLookingForContributors: z.boolean().nullish().optional(),
+  longDescription: z.string().min(1),
+  moreInfo: z.object({
```

Comment:
```suggestion
moreInfo:z.array(
    z.object({
      url: z.string().min(1).nullish(),
      value: z.string().min(1).nullish(),
    })
  ),
```

So you can pass `moreInfo` directly here 

https://github.com/onlydustxyz/marketplace-frontend/blob/f9c2fc749d74e09fc10cea78ff8a3be90c071230/src/pages/ProjectCreation/pages/ProjectInformations/ProjectInformations.tsx#L116